pyramid/httpexceptions.py

```diff
@@ -98,7 +98,9 @@
      a plain-text override of the default ``detail``
 
    ``headers``
-     a list of (k,v) header pairs
+     a list of (k,v) header pairs or a dict to be added to the 
+     response, use content_type='application/json' kwarg to change 
+     the content type of the response.
 
```

Comment:
This needs punctuation to clarify. Previously it seemed that only a list was acceptable. Now it seems that either a list or a dict is acceptable but that only the dict will be added to the response.

Additionally the passage is a fragment and a run-on, and should be broken into a statement and a sentence.

Suggest something like this:

```
A list of (k,v) header pairs or a dict. The headers will be added to the response.
Use the kwarg ``content_type='application/json'`` to change
the content type of the response.
```